### PR TITLE
Fixes for 64-bit.

### PR DESCRIPTION
**Pull request by waddlesplash**
Compiles, but crashes on startup in a dynamic_cast in liblayout. Not sure what's going on there.

### CODE REVIEW

**Comment by stippi** [comment]
Wow, I didn't know there is an x86_64 version of liblayout...

File: common/src/Jamfile [hunk 1]
```diff
@@ -83,7 +83,7 @@ LIBPATHS =  ;
 #	additional paths to look for system headers
 #	thes use the form: #include <header>
 #	source file directories are NOT auto-included here
-SYSTEM_INCLUDE_PATHS = /system/develop/headers/x86/liblayout/
```
Comment (stippi) [inline common/src/Jamfile:86]: Won't this break compiling on x86_ggc2?

**Review by stippi — COMMENTED** [review]
Thanks for the work! I'm surprised that there were only so few changes needed.

File: common/src/Jamfile (hunk 1 above)
Comment (waddlesplash) [reply common/src/Jamfile:86]: No, this was a mistake from when I was testing the GCC7 build on 32-bit. The header directories are theoretically identical.

**Comment by waddlesplash** [comment]
I wonder if the dynamic cast segfault is actually a NULL dereference. Perhaps don't merge this til I have time to investigate whether or not that's actually a WonderBrush bug

**Comment by stippi** [comment]
From what I can tell, the commit contains only valid changes, including the fix for the liblayout system header path. So why not merge it as is?

File: common/src/Jamfile (hunk 1 above)
Comment (stippi) [reply common/src/Jamfile:86]: I misread the change, I thought "/x86/" was added, not removed. Makes sense now.

**Comment by korli** [comment]
Could this change be merged as is please?

**Comment by waddlesplash** [comment]
Yes, I looked into it more and I think it's a liblayout bug. So this can be merged.

**Pull request merged**